layouts/partials/icon.html

```diff
@@ -0,0 +1 @@
+<span class="material-symbols-rounded">{{ . }}</span>
```

Comment:
```suggestion
<span class="material-symbols-rounded" aria-hidden="true">{{ . }}</span>
```
The icons are just decorative, so we should hide them for SR. There is always a label or aria-label to explain the link/button/and so on.